.github/workflows/clippy-check.yml

```diff
@@ -30,8 +37,19 @@ jobs:
           command: lints
           toolchain: nightly
           args: clippy --all-targets --all-features
-      - name: Cargo check
+      - name: Cargo check std
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --release --all-targets --all-features
+      - name: Cargo check no default for non std
         uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --release --all-targets
+          args: --release --no-default-features --features "no_std" --features "borsh_ser"
+      - name: Cargo build no default for non std target
```

Comment:
I think you need some comments explaining what these do, why it seems to be a duplicate of the above and when (or never) should they be removed